test/data.sql

```diff
@@ -615,9 +615,9 @@ INSERT INTO `creditor_group` VALUES
 
 -- Creditors
 INSERT INTO `creditor` VALUES
-  (HUID('42d3756a-7770-4bb8-a899-7953cd859892'), HUID('b0fa5ed2-04f9-4cb3-92f7-61d6404696e7'), 'Personnel'),
+  (HUID('42d3756a-7770-4bb8-a899-7953cd859892'), HUID('b0fa5ed2-04f9-4cb3-92f7-61d6404696e7'), 'Crediteur[Test 2 Patient]'),
```

Comment:
Did these need to change...?

Reply:
Just to respect the formatting of the generate text when creating a creditor

Reply:
Ah, okay.  Makes sense.